test/data/group24.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "build_version_sort": 0,
+    "carry_over_bugrefs": 1,
+    "default_priority": 50,
+    "description": "Responsible maintainer, MAILTO: dummy@example.com.dummy end of line\r\n\r\nTests for openQA within openQA using the test repository [os-autoinst-distri-openQA](https://github.com/os-autoinst/os-autoinst-distri-openQA)\r\n\r\nTesting the packages as provided by [devel:openQA](https://build.opensuse.org/project/show/devel:openQA)",
+    "exclusively_kept_asset_size": 37640732672,
+    "id": 24,
+    "keep_important_logs_in_days": 365,
+    "keep_important_results_in_days": 3650,
+    "keep_logs_in_days": 60,
+    "keep_results_in_days": 90,
+    "name": "openQA",
+    "parent_id": null,
+    "size_limit_gb": 40,
+    "sort_order": 26
```

Comment:
Please delete all keys that we don't care about to make that test data easier to understand.

Reply:
done